src/desktop/package.json

```diff
@@ -170,23 +170,23 @@
     "hw-app-iota": "0.6.1",
     "i18next": "17.0.6",
     "iota.lib.js": "0.5.2",
-    "kdbxweb": "1.2.7",
+    "kdbxweb": "1.5.7",
```

Comment:
`kdbxweb` changed the default encryption rounds in a recent version, so we need to ensure existing SeedVaults are still able to be imported